src/shared/constants/sort-order.constant.ts

```diff
@@ -0,0 +1,9 @@
+export enum SortOrder {
+  Asc = 'ASC',
+  Desc = 'DESC',
+}
+
+export const sortOrderMap = [
+  { key: SortOrder.Asc, name: 'From lower to highest' },
+  { key: SortOrder.Desc, name: 'From highest to lower' },
```

Comment:
Use different names since this will not be used only for prices or ratings but for dates as well. You can simply say 'Ascending' and 'Descending', which is suitable for any case.

Reply:
Fixed